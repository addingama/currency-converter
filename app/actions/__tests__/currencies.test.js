import { getInitialConversion, changeCurrencyAmount } from '../currencies';

it('create a GET_INITIAL_CONVERSION action', () => {
  expect(getInitialConversion()).toMatchSnapshot();
});

it('create a CHANGE_CURRENCY_AMOUNT action', () => {
  expect(changeCurrencyAmount('1000')).toMatchSnapshot();
});
