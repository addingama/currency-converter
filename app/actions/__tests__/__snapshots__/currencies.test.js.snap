// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create a CHANGE_CURRENCY_AMOUNT action 1`] = `
Object {
  "amount": 1000,
  "type": "CHANGE_CURRENCY_AMOUNT",
}
`;

exports[`create a GET_INITIAL_CONVERSION action 1`] = `
Object {
  "type": "GET_INITIAL_CONVERSION",
}
`;
